lib/odoobaselib/__init__.py

```diff
@@ -35,6 +35,12 @@
 logging.root.setLevel(_log_level)
 
 
+def addons_active():
+    """ Returns a list of active addons, as defined in `addons.yml`. """
```

Comment:
Please remove the surrounding spaces in the docstring, following [PEP-0257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).